templates/news/update.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+<h1>{% if news.pk %}{% translate "Edit News" %}{% else %}{% translate "Create News" %}{% endif %}</h1>
```

Comment:
What's "translate" ?

Reply:
`{% translate %}` is [the Django template tag](https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#translate-template-tag) that allows us to mark literal strings for potential/future translation. I'm not sure if this is something we'd want or not, it certainly does not hurt nor force us to translate anything yet, but if there are future plans to show the site in other languages, this is the way.

Reply:
oh cool